README.md

```diff
@@ -1,2 +1,23 @@
-# AndroidCommons
-AndroidCommons
+# Android Commons
+A library for faster development of android applications by Cotta & Cush Ltd.
+
+# Getting Started
+```
+    Using gradle:
+
+    dependencies {
+        compile 'com.github.CottaCush:AndroidCommons:-SNAPSHOT'
+    }
+
+    allprojects {
+        repositories {
+            maven {url 'https://jitpack.io'}
+        }
+    }
+
+    This library requires build tools version 25
+
+```
+
+# Versioning
+We use [SemVer](http://semver.org/) for versioning.
```

Comment:
Change this to "Semver is used for versioning", write README in reported speech.